doc/releases/changelog-dev.md

```diff
@@ -273,6 +273,10 @@
 
 <h3>Bug fixes</h3>
 
+* Fix a bug in the extrapolation part of the `catalyst.mitigate_with_zne` function that would lead
```

Comment:
nit: the sentence reads a bit redundant. A bug in extrapolation code leads to..incorrect extrapolated result"

Reply:
```suggestion
* Fix a bug in `catalyst.mitigate_with_zne` that would lead
```